### DISCUSSION TITLE
Better demonstration of Filter/Permission rules

### DISCUSSION THREAD

**Discussion by mdav43** [directus/directus#6614] [Archive]
Hi, I've set up a test instance of Directus and super impressed at its simplicity and functionality at the same time.

One issue I have is setting up specific permissions/filters.

To simplify, I have one single table. I have 4 distinct users (e.g. USER-A,USER-B,USER-C,USER-D) and each user has the following roles/group (OFFICE-A,OFFICE-B,OFFICE-C,HEADOFFICE).

Through the web interface, how can I set the table so that USER-A can only items created by users in the OFFICE-A group/role OFFICE-A items etc etc and USER-D is able to see all items in the table?

I haven't see a guide anywhere even though in the documentation it seems possible.

**Comment by benhaynes** [comment]
Have you looked into setting permissions for an `owner` or `creator` field that holds the user/role? You can have that field automatically save the user that created/updated the item. Then filter against the `$CURRENT_USER` and/or `$CURRENT_ROLE`.

https://docs.directus.io/reference/filter-rules/#dynamic-variables
